scripts/pull_request_blaster_outer/new_ruby_api_service.rb

```diff
@@ -0,0 +1,157 @@
+#! /usr/bin/env ruby
+
+missing = %w[GITHUB_API_TOKEN CODECLIMATE_API_TOKEN DEPLOY_SERVER DEPLOY_NAMESPACE DEPLOY_IMAGE DEPLOY_TOKEN DEPLOY_USER].reject { |k| ENV.key?(k) }
+if missing.any?
+  puts "ERROR: Expected the following env vars set:\n\t#{missing.join("\n\t")}"
+  exit 1
+end
+
+#
+# Travis
+#
+
+require 'fileutils'
+
+puts "Enabling repo..."
+`travis login --github-token $GITHUB_API_TOKEN`
+`travis enable --org`
+
+puts "Setting travis env vars..."
+`travis env set DEPLOY_SERVER    '#{ENV["DEPLOY_SERVER"]}'`
+`travis env set DEPLOY_NAMESPACE '#{ENV["DEPLOY_NAMESPACE"]}'`
+`travis env set DEPLOY_IMAGE     '#{ENV["DEPLOY_IMAGE"]}'`
+`travis env set DEPLOY_TOKEN     '#{ENV["DEPLOY_TOKEN"]}'`
+`travis env set DEPLOY_USER      '#{ENV["DEPLOY_USER"]}'`
+
+puts "Creating .travis.yml file..."
+File.write(".travis.yml", <<~EOF)
+  ---
+  language: ruby
+  sudo: required
+  cache: bundler
+  rvm:
+  - 2.4.5
+  services:
+  - docker
+  addons:
+    postgresql: '9.5'
+  env:
+    global:
+    - DATABASE_USERNAME=root
+    - DATABASE_PASSWORD=smartvm
+  before_install:
+  - gem install bundler
+  - psql -c "CREATE USER $DATABASE_USERNAME SUPERUSER PASSWORD '$DATABASE_PASSWORD';" -U postgres
+  before_script:
+  - curl -L https://codeclimate.com/downloads/test-reporter/test-reporter-latest-linux-amd64 > ./cc-test-reporter
+  - chmod +x ./cc-test-reporter
+  - "./cc-test-reporter before-build"
+  - bundle exec rake db:create db:migrate
+  after_script:
+  - "./cc-test-reporter after-build --exit-code $TRAVIS_TEST_RESULT"
+  deploy:
+    provider: script
+    script: bash .travis/deploy.sh
+    on:
+      branch: master
+EOF
```

Comment:
I'm not sure what to think about having a vanilla .travis.yml embedded here.  I guess we can adjust it as we see fit but it feels like something that could grow stale.